<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DataRecordWidget</class>
 <widget class="QWidget" name="DataRecordWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>637</width>
    <height>477</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="page_record">
      <layout class="QGridLayout" name="gridLayout_3">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item row="0" column="0">
        <widget class="QSplitter" name="splitter_data_record">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="handleWidth">
          <number>2</number>
         </property>
         <widget class="QWidget" name="widget_chart" native="true">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>300</height>
           </size>
          </property>
         </widget>
         <widget class="QWidget" name="widget_data_record_handle" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <layout class="QGridLayout" name="gridLayout_2">
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <property name="horizontalSpacing">
            <number>0</number>
           </property>
           <property name="verticalSpacing">
            <number>6</number>
           </property>
           <item row="0" column="0">
            <widget class="QTableWidget" name="tableWidget_data_record">
             <property name="alternatingRowColors">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <spacer name="horizontalSpacer">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QToolButton" name="toolButton_pre_day">
               <property name="text">
                <string>前一天</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QToolButton" name="toolButton_pre_page">
               <property name="text">
                <string>上一页</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QDateEdit" name="dateEdit_date_page">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="displayFormat">
                <string>yyyy-MM-dd</string>
               </property>
               <property name="calendarPopup">
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string>页码:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="spinBox_data_page">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="minimum">
                <number>1</number>
               </property>
               <property name="maximum">
                <number>6</number>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QToolButton" name="toolButton_next_page">
               <property name="text">
                <string>下一页</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QToolButton" name="toolButton_next_day">
               <property name="text">
                <string>后一天</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_2">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton_day">
               <property name="text">
                <string>全天</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton_db_page">
               <property name="text">
                <string>管理数据库</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_db_handle">
      <layout class="QGridLayout" name="gridLayout_5">
       <item row="0" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <layout class="QGridLayout" name="gridLayout_4">
           <item row="1" column="0">
            <widget class="QCheckBox" name="checkBox_db_ip">
             <property name="text">
              <string>设备IP:</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QDateEdit" name="dateEdit_db_date">
             <property name="displayFormat">
              <string>yyyy/MM/dd</string>
             </property>
             <property name="calendarPopup">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="2" column="3">
            <widget class="QPushButton" name="pushButton_data_clear">
             <property name="text">
              <string>清空数据记录</string>
             </property>
            </widget>
           </item>
           <item row="1" column="4">
            <widget class="QPushButton" name="pushButton_alarm_out">
             <property name="text">
              <string>导出报警记录</string>
             </property>
            </widget>
           </item>
           <item row="1" column="2">
            <widget class="QPushButton" name="pushButton_date_record">
             <property name="text">
              <string>数据记录</string>
             </property>
            </widget>
           </item>
           <item row="2" column="2">
            <widget class="QPushButton" name="pushButton_alarm_record">
             <property name="text">
              <string>告警记录</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLineEdit" name="lineEdit_db_ip">
             <property name="placeholderText">
              <string>127.0.0.1</string>
             </property>
            </widget>
           </item>
           <item row="1" column="3">
            <widget class="QPushButton" name="pushButton_data_out">
             <property name="text">
              <string>导出数据记录</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QCheckBox" name="checkBox_db_date">
             <property name="text">
              <string>日期:</string>
             </property>
            </widget>
           </item>
           <item row="2" column="4">
            <widget class="QPushButton" name="pushButton_alarm_clear">
             <property name="text">
              <string>清空报警记录</string>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QCheckBox" name="checkBox_db_mac">
             <property name="text">
              <string>设备MAC:</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QLineEdit" name="lineEdit_db_mac">
             <property name="placeholderText">
              <string>00:50:56:92:7A:42</string>
             </property>
            </widget>
           </item>
           <item row="3" column="2" colspan="2">
            <widget class="QLabel" name="label">
             <property name="text">
              <string>分隔符可为:(&quot;:&quot;  &quot;-&quot; 空格 )</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="horizontalSpacer_6">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item row="3" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_9">
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_up">
           <property name="text">
            <string>上一页</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_10">
           <item>
            <widget class="QLabel" name="label_21">
             <property name="text">
              <string>第</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_current">
             <property name="text">
              <string>*</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_22">
             <property name="text">
              <string>页</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QLabel" name="label_23">
           <property name="text">
            <string>/</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_11">
           <item>
            <widget class="QLabel" name="label_24">
             <property name="text">
              <string>共</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_sum">
             <property name="text">
              <string>*</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_25">
             <property name="text">
              <string>页</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_down">
           <property name="text">
            <string>下一页</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_4">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_data_record_page">
           <property name="text">
            <string>返回数据记录页面</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="2" column="0">
        <widget class="QTableWidget" name="tableWidget_db"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
